index.bs

```diff
@@ -4902,6 +4991,55 @@ as candidates to be employed in a [=registration ceremony=].
 : Authenticator extension output
 :: None.
 
+
+## Credential Properties Extension (credProps) ## {#sctn-authenticator-credential-properties-extension}
+
+This extension allows [=[WRPS]=] to query additional properties of the authenticator's credential created during a 
+[=registration ceremony=] or used during an [=authentication ceremony=].
+
+: Extension identifier
+:: `credProps`
+
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|Authentication=]
+
+: Client extension input
+:: The Boolean value [TRUE] to indicate that this extension is requested by the [=[RP]=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+        boolean credProps;
+    };
+    </xmp>
+
+: Client extension processing
+:: None, except creating the authenticator extension input from the client extension input.
+
+: Client extension output
+:: Returns the dictionary of discovered credential properties as reported by either the authenticator itself or by the 
```

Comment:
I don't think it's necessary to have the client forward the authenticator extension outputs, since those are already returned in the authenticator data. Maybe something like this?

>Returns the credential properties as discovered by the client, if any, such as when a platform authenticator is used. The client MAY omit any properties at its discretion, and SHOULD omit any properties that would be duplicated in the authenticator extension output. This dictionary is initially defined [...]

Reply:
Agreed - fixed in subsequent commit.